frontend/packages/kubevirt-plugin/src/components/LabelsList/labels-list.tsx

```diff
@@ -22,26 +22,33 @@ export const LabelsList = <T extends IDLabel = IDLabel>({
   onLabelAdd,
   onLabelChange,
   onLabelDelete,
+  extraColumn,
   addRowText = ADD_LABEL,
   emptyStateAddRowText = ADD_LABEL,
 }: LabelsListProps<T>) => (
   <>
     <Grid className="kv-labels-list__grid">
       {labels.length > 0 && [
         <React.Fragment key="label-title-row">
-          <GridItem span={6}>
+          <GridItem span={extraColumn ? 4 : 6}>
             <Text component={TextVariants.h4}>{LABEL_KEY}</Text>
           </GridItem>
-          <GridItem span={6}>
+          <GridItem span={extraColumn ? 4 : 6}>
             <Text component={TextVariants.h4}>{LABEL_VALUE}</Text>
           </GridItem>
+          {extraColumn && (
+            <GridItem span={4}>
+              <Text component={TextVariants.h4}>{extraColumn.title}</Text>
+            </GridItem>
+          )}
         </React.Fragment>,
         labels.map((label) => (
```

Comment:
we probably should allow adding children instead for extensibility and map this outside